web/src/lib/api/types/pipeline.ts

```diff
@@ -98,31 +98,31 @@ export type BuildStatus =
   | 'started'
   | 'success';
 
-export type BuildProc = {
+export type PipelineProc = {
   id: number;
-  build_id: number;
+  pipeline_id: number;
```

Comment:
does this work as we did no change json schema?

Reply:
It wont in JS var names are json keys as well.

Reply:
:point_up_2: so we have to rename json fields here ...